narwhal/executor/src/subscriber.rs

```diff
@@ -217,7 +220,14 @@ impl<Network: SubscriberNetwork> Fetcher<Network> {
             let output_cert = cert.clone();
 
             // TODO(metrics): Set `subscriber_current_round` to `cert.round() as i64`.
```

Comment:
Yeah, I didn't think this would be an important metric since we have other ways to get the current round and I didn't wanna spam too many metrics. For example, the metric for certificate latency has the round attached. What do you think?

Reply:
I don't mind so long as it's deliberate—seems reasonable though 👍